cogeo_mosaic/create.py

```diff
@@ -58,10 +53,10 @@ def create_mosaic(
     if not quiet:
         click.echo("Get files footprint", err=True)
 
-    results = get_footprints(dataset_list, max_threads=max_threads, quiet=quiet)
+    features = get_footprints(dataset_list, max_threads=max_threads, quiet=quiet)
```

Comment:
👍

Reply:
What I was saying before is that now this function is essentially just a wrapper to call `get_footprints`, do some error checking, and pass `features` on.